src/main/java/microblogevolved/Message.java

```diff
@@ -0,0 +1,63 @@
+package microblogevolved;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Created by darionmoore on 12/17/16.
+ */
+public class Message {
+    private String message;
+    private static Integer ID;
+    private int createdAt;
+    private static ArrayList<Message> messages = new ArrayList();
+
+    public Message(String message, Integer ID, int createdAt) {
+        this.message = message;
+        this.ID = ID;
+        this.createdAt = createdAt;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public int getID() {
+        return ID;
+    }
+
+    public void setID(Integer ID) {
+        this.ID = ID;
+    }
+
+    public int getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(int createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    static void messageId() {
```

Comment:
I don't understand what this method does.  You are messing with a static id on the class.  Not only that the loop simply counts how many messages x2 since it does ++ which keeps changing the ids.  It isn't at all looking for the message's id so you can't get the max id.